service/scheduler.go

```diff
@@ -80,29 +80,37 @@ func (s *Scheduler) lockAndRun(db *gorm.DB, wf table.Workflow) {
 		APIKey:     s.OrchardAPIKey,
 	}
 
-	orchardID, err := client.Create(wf)
-	scheduleStatus := "error"
+	scheduleStatus := make(map[string]string)
+	orchardIDs, err := client.Create(wf)
 	if err != nil {
 		notifyOwner(wf, err)
+		scheduleStatus[""] = "error"
 	} else {
-		err = client.Activate(orchardID)
-		if err != nil {
-			fmt.Println(err)
-			notifyOwner(wf, err)
+		for _, orchardID := range orchardIDs {
+			err = client.Activate(orchardID)
+			if err != nil {
+				fmt.Println(err)
+				notifyOwner(wf, err)
+				scheduleStatus[orchardID] = "error"
+			} else {
+				scheduleStatus[orchardID] = "activated"
```

Comment:
do we still want to activate the other workflows when there is error in one of the workflows?

Reply:
@anniezhang77 yes, the next functionality to add is rollback when encountering errors

Reply:
rollback added